src/lib/Libecl/ecl_verify.c

```diff
@@ -663,9 +664,10 @@ get_attr_type(struct ecl_attribute_def attr_def)
 	/*
 	 * Consider an attr def public if it has any of the write flags set
 	 */
-	if (attr_def.at_flags & (ATR_DFLAG_USWR | ATR_DFLAG_OPWR | ATR_DFLAG_MGWR))
+	if (attr_def.at_flags & (ATR_DFLAG_SvWR | ATR_DFLAG_SvRD))
+		return TYPE_ATTR_INVISIBLE;
```

Comment:
@bhroam , thanks for catching that, it's definitely weird, but I guess it is the way it is, I've put it as the 3rd case now, please let me know if this works. Thanks.